whats-that-typeclass-functor/article.md

```diff
@@ -0,0 +1,467 @@
+You might already know the `map` function for lists:
+
+```haskell
+> map (+1) [1, 2, 3]
+[2, 3, 4]
+```
+
+It takes a function of type `a -> b` and applies it to each element of a list of type `[a]`. The values change, but the context (`[]`) remains the same.
```

Comment:
> The values change, but the context (`[]`) remains the same.

This seems a bit confusing. May be we can start by saying some types can have other types inside of them, and a `map` operation acts on the inner type, leaving the outer type the same, or something like that. This notion of a 'context' might be a bit jarring for a beginner.

Reply:
It does mention that it is for lists.

Reply:
I've edited this sentence a bit and added a note about context

I also believe there are a lot of articles about monads calling them context. However, it's not some formal notation or documented thing, but the most suitable word explaining what monads do. The same for functor, it's just a word that helps to grasp the idea. So replacing it with another one less appropriate, that doesn't resonate with monads explanation, looks to me like just comlicating the things

Reply:
Love the rewording!